### PR TITLE
Support for completion pages in activities.

### PR DESCRIPTION
**Pull request by knowuh**
Give authors the ability to add a 'Completion' page.  This page lets students know that they have finished their work, and gives them the option to move on to the next activity in a sequence, print their work, or exit the activity.

This feature is described in this PT story:

https://www.pivotaltracker.com/story/show/59366968

### CODE REVIEW

**Comment by knowuh** [comment]
@scytacki  Could you look at this?  If you are swamped I can try other folks.

**Comment by knowuh** [comment]
I think this is as close to the mockups as I can get for now.  @scytacki can you review?

File: app/views/interactive_pages/_show_completion.html.haml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  complete_header_t   = t("COMPLETION_PAGE.HEADER", {activity: @activity.name})
+  congratulations_t   = t("COMPLETION_PAGE.CONGRATULATIONS")
+  finished_activity_t = t('COMPLETION_PAGE.FINISHED_ACTIVITY', activity: @activity.name)
+  not_finished_activity_t = t('COMPLETION_PAGE.UNFINISHED_ACTIVITY', activity: @activity.name)
+
+  finished_sequence_t = t('COMPLETION_PAGE.FINISHED_SEQUENCE')
+  generate_report_t   = t("COMPLETION_PAGE.GENERATE_A_REPORT")
+  start_next_t        = t("COMPLETION_PAGE.START_NEXT")
+  exit_label_t        = t("COMPLETION_PAGE.EXIT_LABEL")
+  next_up             = t("COMPLETION_PAGE.NEXT_UP")
```
Comment (scytacki) [inline app/views/interactive_pages/_show_completion.html.haml]: for consistency this looks like it should have a `_t` too.

File: config/locales/en.yml [hunk 2]
```diff
@@ -114,4 +114,17 @@ en:
...
+    FINISHED_ACTIVITY: "You have reached the end of \"%{activity}\" and your work has been saved."
+    UNFINISHED_ACTIVITY: "It looks like you haven't quite finished \"%{activity}\" yet. The answers you've given have been saved."
+    FINISHED_SEQUENCE: "You have completed all your work for this module!"
+    CONTINUE: "Continue on to %{next_activity}, or click \"Exit\" to close the activity."
+    EXIT: "Your work has been saved. Click \"Exit\" to close the activity."
+    START_NEXT: "Start Next Activity"
+    GENERATE_A_REPORT: "Show All Answers"
+    CONFIRM_CLOSE: "Exit %{activity}?"
+    EXIT_LABEL: "Exit"
+    NEXT_UP: "Next Up…"
```
Comment (scytacki) [inline config/locales/en.yml]: would be better to add a return here so future changes don't appear to be modifying the last line

**Comment by scytacki** [comment]
Generally looks good. 👍  I added a few niggly line comments.

**Comment by knowuh** [comment]
@scytacki Thanks for the review.  Small fixes added in 12532ef

**Pull request merged**